packages/logger/test/e2e/logger/workerLogger.js

```diff
@@ -1,9 +1,10 @@
 import fs from "node:fs";
 import worker from "node:worker_threads";
+// eslint-disable-next-line import/no-extraneous-dependencies
 import {expose} from "@chainsafe/threads/worker";
 
 const parentPort = worker.parentPort;
-const workerData = worker.workerData as {logFilepath: string};
+const workerData = worker.workerData;
```

Comment:
Have to convert this TS file to JS as `Worker` can only load the JS files. Earlier this was handled by runtime compilation with global node loader.